### PR TITLE
Some fixups

### PR DESCRIPTION
**Pull request by andrunko**
https://phabricator.endlessm.com/T28780

### CODE REVIEW

File: js/ui/appDisplay.js [hunk 1]
```diff
@@ -624,6 +624,7 @@ class AppDisplay extends BaseAppView {
                         return;
                     }
                     icon = new FolderIcon(item, this);
+                    icon.connect('clicked', () => { this._bgAction.release(); })
```
Comment (GeorgesStavracas) [inline js/ui/appDisplay.js]: Style issue, the correct style should be:

```javascript
icon.connect('clicked', () => this._bgAction.release());
```

File: js/ui/appDisplay.js [hunk 2]
```diff
@@ -636,6 +637,7 @@ class AppDisplay extends BaseAppView {
                 icon = new AppIcon(app, {
                     isDraggable: favoritesWritable,
                 });
+                icon.connect('clicked', () => { this._bgAction.release(); })
```
Comment (GeorgesStavracas) [inline js/ui/appDisplay.js]: Ditto

File: js/ui/appDisplay.js (hunk 1 above)
Comment (andrunko) [reply js/ui/appDisplay.js]: Updated.

File: js/ui/appDisplay.js (hunk 2 above)
Comment (andrunko) [reply js/ui/appDisplay.js]: Updated.

**Pull request merged**